prov/efa/src/efa_device.c

```diff
@@ -130,6 +130,27 @@ int efa_device_init(void)
 	return ret;
 }
 
+bool efa_device_support_rdma_read(void)
+{
+#ifdef HAVE_RDMA_SIZE
+	int err;
+	struct efadv_device_attr efadv_attr;
+
+	fprintf(stderr, "in efa_device_support_rdma_read(). dev_cnt: %d\n", dev_cnt);
+	if (dev_cnt <=0)
```

Comment:
Where are you querying dev_cnt? Did you meant to call `ibv_get_device_list` here?

Reply:
dev_cnt is a global variable initialized in efa_device_init(). efa_device_init() was called by init_lower_efa_provider(), which is called by rxr_ini(). So it should have been called when efa_device_support_rdma_read() was called.